cranelift-codegen/src/legalizer/mod.rs

```diff
@@ -498,3 +572,67 @@ fn expand_stack_store(
     mflags.set_aligned();
     pos.func.dfg.replace(inst).store(mflags, val, addr, 0);
 }
+
+/// Split a load into two parts before `iconcat`ing the result together.
+fn narrow_load(
```

Comment:
Would it make sense to call this `split_load`?

Reply:
This is in the narrow legalization group.

Reply:
Ah, makes sense.